metrics/micrometer-metrics/src/main/java/org/apache/iotdb/metrics/micrometer/reporter/MicrometerPrometheusReporter.java

```diff
@@ -65,11 +65,10 @@ public boolean start() {
             .route(
```

Comment:
Last month, there is a feature merged into master, it's about providing rest apis for users.
The rest service will also startup a http server, can these two http servers merged into one?

Reply:
ok, it's a good idea. I think we could finish this in another PR.

Reply:
We will do this on this issue: https://issues.apache.org/jira/projects/IOTDB/issues/IOTDB-2127?filter=reportedbyme